### DISCUSSION TITLE
Add option display trailing decimal placeholder if integer

### DISCUSSION THREAD

**Discussion by rfikes4** [barvian/number-flow#77] [Help] [answered]
Currently if the value is `90.0` it renders as `90` and remove the `.0`. There should be a format option to keep the trailing `.000`

**Comment by barvian** [comment, marked as answer]
Hi 👋, have you seen [`minimumFractionDigits`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumfractiondigits)? It should help with this. [Sandbox](https://codesandbox.io/p/sandbox/number-flow-77-7rr9vv?workspaceId=cbc394d8-76be-4eee-b6ad-557eaf0d663f)

**Reply by rfikes4** [reply]
Woo thank you!!! Quick response!
